cookbook/deployment/platformsh.rst

```diff
@@ -37,9 +37,11 @@ Platform.sh how to deploy your application (read more about
 
     # The name of this app. Must be unique within a project.
     name: myphpproject
-
-    # The toolstack used to build the application.
-    toolstack: "php:symfony"
+    
+    # The type of the application to build.
+    type: php:5.4
```

Comment:
if it does, I would vote to use a newer version if possible (although no PHP is used by this project, it's quite nice to show a good practice not using versions that reached eol)